src/main/cpp/mechanisms/IntakeManager/OffState.cpp

```diff
@@ -77,6 +77,6 @@ bool OffState::AtTarget()
 bool OffState::IsTransitionCondition ( bool considerGamepadTransitions )
 {
 	// To get the current state use m_mechanism->GetCurrentState()
-	return false;
-	// return (considerGamepadTransitions && TeleopControl::GetInstance()->IsButtonPressed(TeleopControlFunctions::EXAMPLE_MECH_FORWARD));
+
+	return (considerGamepadTransitions && (TeleopControl::GetInstance()->IsButtonPressed(TeleopControlFunctions::INTAKE) && m_mechanism->GetIntake()) || (m_mechanism->STATE_EXPEL && TeleopControl::GetInstance()->IsButtonPressed(TeleopControlFunctions::EXPEL)) || (m_mechanism->GetIntakeSensorState() && (m_mechanism->STATE_PROCESS || m_mechanism->STATE_HOLD)));
```

Comment:
Need to add Nots(!) to button press and sensor states, we need to enter this state if the button isn't pressed anymore and we don't have a algae

Also you need to check current state and see if the state is process or hold. 

Return true if

Current State is Intake AND Intake Button NOT Pressed
Current State is Expel AND Expel Button NOT Pressed
Intake Sensor = False AND current state is Process OR Hold